### DISCUSSION TITLE
blog/jwt-authentication-in-golang/

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#10] [General]
# blog/jwt-authentication-in-golang/

In this article, we will learn about implementing JWT Authentication in Golang REST APIs and securing it with Authentication Middleware. We will be building a simple, yet neatly organized Golang REST API with packages like Gin for Routing (mostly), GORM for persisting user data to a MySQL Database, and so on.

https://codewithmukesh.com/blog/jwt-authentication-in-golang/

**Comment by swapnilkant11** [comment]
The article is really well explained, just a suggestion that It would look more good and easy to debug if you have added code snippets as well to figure out the line number for understanding the code
Thanks

**Comment by swapnilkant11** [comment]
Also trying using postman for sending requests to your API, it's easy to use and improves readability

**Comment by TPAlves** [comment]
Thank you very much
